test/unix/main.ml

```diff
@@ -273,6 +273,28 @@ module Readonly = struct
     Index.close rw;
     Index.close ro
 
+  (* check that the ro instance is "snapshot isolated", e.g. it can read old
+     values, even when rw overwrites them. *)
+  let readonly_snapshot () =
+    let Context.{ rw; clone; tbl } = Context.full_index () in
+    let ro = clone ~readonly:true () in
+    check_equivalence rw tbl;
+    check_equivalence ro tbl;
+    let tbl2 =
+      let h = Hashtbl.create 103 in
```

Comment:
```suggestion
      let h = Hashtbl.create 0 in
```

I'm not sure why we picked 103 as the default size...

Reply:
103 is a good prime number :p